containers/podman-desktop
# Feat: add dropshadows

### What does this PR do?

Adds dropshadows to the main nav, the settings nav, and an inner shadow highlight to the top of the window just under the titlebar.

### Screenshot/screencast of this PR

![Screenshot from 2023-05-01 20-36-33](https://user-images.githubusercontent.com/799683/235555306-7b728d48-3b42-4a95-999f-3286e284bd45.png)
![Screenshot from 2023-05-01 20-36-23](https://user-images.githubusercontent.com/799683/235555311-d09e22cd-1cc1-4831-93a3-39193f2fbf9e.png)
![Screenshot from 2023-05-01 20-36-17](https://user-images.githubusercontent.com/799683/235555313-a99f611e-899e-4203-8a0e-0fdf9115bf5a.png)


### What issues does this PR fix or reference?

supports https://github.com/containers/podman-desktop/issues/2287

### How to test this PR?

Try the main nav, the settings nav, and various form dialogs... make sure the drop shadows show up along the left side, particularly on long-scrolling screens.
